docs/source/howto/plugins_develop.rst

```diff
@@ -18,26 +18,24 @@ Creating a plugin package
 =========================
 
 
-AiiDA plugins can be bundled and distributed in a `Python package <packages>`_ that provides a set of extensions to AiiDA.
+AiiDA plugins can be bundled and distributed in a `Python package <https://docs.python.org/3/tutorial/modules.html?highlight=package#packages>`_ that provides a set of extensions to AiiDA.
 
 .. note::
 
   The Python community uses the term 'package' rather loosely.
-  Depending on context, it may refer simply to a folder containing individual Python modules or it may include the files necessary for building and installing a package to be distributed via the `Python Package Index (PyPI) <pypi>`_.
-
-.. _packages: https://docs.python.org/2/tutorial/modules.html?highlight=package#packages
-
+  Depending on context, it may refer simply to a folder containing individual Python modules or it may include the files necessary for building and installing a package to be distributed via the `Python Package Index (PyPI) <https://pypi.python.org>`_.
```

Comment:
you don't need replaces these with explicit links, they were just missing an underscore after the reference: `` `Python Package Index (PyPI) <pypi_>`_ ``

Reply:
Oh! yes, I google around and didn't find this :rofl: I'll change them back and adding the underscore.

Reply:
Yeh I only know it, because I literally re-wrote most of the RST parser recently lol: https://github.com/executablebooks/rst-to-myst/blob/main/rst_to_myst/inliner.py